presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -444,6 +444,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 * Using partitioning(``partitioned_by``) or bucketing(``bucketed_by``) columns are not supported in ``CREATE TABLE``.
 * ``ALTER TABLE`` commands modifying columns are not supported.
 
+Hive Connector Procedures
+---------------------------
+
+.. function:: system.create_empty_partition(schema, table, partition_cols, partition_vals)
```

Comment:
We actually shouldn't use `function` here, since these aren't SQL functions and aren't part of the global namespace (the system connector documentation should also be updated). Maybe we should format it like we do for the tables list in the system connector. Or perhaps we should leave this section out and just rely on the example?

Reply:
@electrum : Table list looks really strange:

<img width="1227" alt="screen shot 2018-11-05 at 10 50 36 pm" src="https://user-images.githubusercontent.com/799346/48047574-4cadf380-e14d-11e8-9ed3-5c237c90a7af.png">


I think we should still list it. Maybe function is not the best choice. Let me test...

Reply:
What about just use "``":

<img width="1019" alt="screen shot 2018-11-05 at 10 54 58 pm" src="https://user-images.githubusercontent.com/799346/48047738-e675a080-e14d-11e8-810f-0d5dfea56e35.png">

Reply:
Hmm, I see what you mean. I think we can do better, but this is good for now.